src/main/java/com/gd/clinic/entities/Patient.java

```diff
@@ -0,0 +1,53 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Patients")
+public class Patient{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    private String name;
+
+    @NonNull
+    private String diagnosis;
+
+    @NonNull
+    private String insuranceNumber;
+
+    @Type(type = "uuid-char")
+    private UUID doctorId;
+
+    private String status;
```

Comment:
this should be enum

Reply:
This field's been declared as Enum and uses the values from EventResponseDto